proto/eth/v2/blobs.proto

```diff
@@ -39,6 +39,7 @@ message BlobSidecar {
         bytes kzg_proof = 8 [(ethereum.eth.ext.ssz_size) = "48"];
 }
 
+// depreciated: TODO: remove this when denebblockcontents is updated
```

Comment:
Let's chnage this TODO since `denebblockcontents` is not the item left

```suggestion
// TODO: remove after blob sidecar inclusion proof format is used
```